ReactAndroid/src/main/java/com/facebook/react/modules/netinfo/NetInfoModule.java

```diff
@@ -81,25 +81,21 @@ public String getName() {
   }
 
   @ReactMethod
-  public void getCurrentConnectivity(Callback successCallback, Callback errorCallback) {
+  public void getCurrentConnectivity(Promise promise) {
     if (mNoNetworkPermission) {
-      if (errorCallback == null) {
-        FLog.e(ReactConstants.TAG, MISSING_PERMISSION_MESSAGE);
```

Comment:
Oops, this is not a class specific TAG it is the general TAG we use all over.  Never mind.